host/CMakeLists.txt

```diff
@@ -483,7 +483,10 @@ if (OE_SGX)
       set_target_properties(sgx_dcap_ql PROPERTIES IMPORTED_LOCATION
                                                    ${LIBSGX_QE})
     endif ()
-    target_link_libraries(oehost PUBLIC $<BUILD_INTERFACE:sgx_dcap_ql>)
+    # target_link_libraries(oehost PUBLIC $<BUILD_INTERFACE:sgx_dcap_ql>)
```

Comment:
Why is there a commented out line?   Either remove it, uncomment it out, or add a comment explaining why it's commented out and when the developer should enable it.

Reply:
Fixed.